.spectral.yml

```diff
@@ -1,4 +1,4 @@
 extends:
```

Comment:
@MelvLee Graag even checken of de master branch versie niet de correcte is.

Reply:
klopt. Dat is nu gefixt met een merge vanuit main